src/ThreeEditor/js/Viewport.js

```diff
@@ -238,10 +238,40 @@ export function Viewport(
 
     });
 
+    window.addEventListener('keydown', function (event) {
```

Comment:
Is it a new behavior ?

Reply:
Now shift enables snapping on when transforming

Reply:
this feature with shift works nicely